pig/src/main/resources/interpreter-setting.json

```diff
@@ -15,6 +15,12 @@
         "propertyName": "zeppelin.pig.includeJobStats",
         "defaultValue": "false",
         "description": "flag to include job stats in output"
+      },
+      "zeppelin.pig.execType": {
+        "envName": "SPARK_MASTER",
+        "propertyName": "SPARK_MASTER",
```

Comment:
`SPARK_MASTER` would only take effect when `execType` is set as spark. Tez will ignore `SPARK_MASTER`. This is internal mechanism of pig.